fsck.c

```diff
@@ -1323,6 +1337,62 @@ int fsck_finish(struct fsck_options *options)
 	return ret;
 }
 
+__attribute__((format (printf, 4, 5)))
+int fsck_refs_report(struct fsck_refs_options *o,
```

Comment:
There is a lot of duplication between this function and `report()`. It would be nice if we could reduce the duplication to a bare minimum. If we had a common `fsck_options` structure, then this would be easier to realize.

Reply:
After discussing offline, currently, it's hard for us to extract some common logics. Will handle this later.